static/scss/error.scss

```diff
@@ -0,0 +1,62 @@
+// sass-lint:disable mixins-before-declarations
+.error-block {
+  min-height: calc(100vh - 98px);
+  background: url(#{$static-path}/images/transparent-pixels-bg.png) repeat-x 50% 100%;
+  border-top: 1px solid #000;
```

Comment:
can we use descriptive name of color instead of code ?

Reply:
Updated